applications/ContactStructuralMechanicsApplication/python_scripts/adaptative_remeshing_contact_structural_mechanics_analysis.py

```diff
@@ -22,158 +27,133 @@ def __init__(self, model, project_parameters):
         # Construct the base analysis.
         default_params = KM.Parameters("""
         {
-            "max_iteration" : 1,
-            "analysis_type" : "linear",
+            "max_iteration"    : 1,
+            "analysis_type"    : "linear",
             "contact_settings" :
             {
                 "fancy_convergence_criterion" : false
```

Comment:
I will suggest to remove the prefix `fancy_`

Reply:
It is alrady in my app, not related with this PR (is not in red or green)

Reply:
we have the `KRATOS_COLORED_OUTPUT` definition in the meantime which should be used for this

Reply:
@loumalouomega its about usability, it not common to have `fancy_` as a prefix also it is not explaining anything. my be with the suggestion of philipp you can altogether remove this setting.
of course you can choose any name you want for the key as I put it there, its a suggestion !

Reply:
> 
> 
> we have the `KRATOS_COLORED_OUTPUT` definition in the meantime which should be used for this

This is not related with this PR in any case, and is not the same as the colours, is the structure I use table like